stdlib/public/Darwin/Foundation/JSONEncoder.swift

```diff
@@ -1205,7 +1194,7 @@ open class JSONDecoder {
     open func decode<T : Decodable>(_ type: T.Type, from data: Data) throws -> T {
         let topLevel: Any
         do {
-           topLevel = try JSONSerialization.jsonObject(with: data)
+           topLevel = try JSONSerialization.jsonObject(with: data, options: [.fragmentsAllowed])
```

Comment:
Is there a reason to why `options` is not configurable? I'd imagine there are some use-cases where you do not want to allow decoding fragments.